src/sequelize.js

```diff
@@ -1280,7 +1280,8 @@ Sequelize.prototype.Validator = Sequelize.Validator = Validator;
 
 Sequelize.Model = Model;
 
-Sequelize.QueryInterface = AbstractQueryInterface;
+const QueryInterface = AbstractQueryInterface;
+Sequelize.QueryInterface = QueryInterface;
```

Comment:
So in my opinion the only change that's needed (out of the several that are now done in this PR) is;
```suggestion
Sequelize.AbstractQueryInterface = AbstractQueryInterface;
```